NDTensors/test/Project.toml

```diff
@@ -16,6 +16,4 @@ TBLIS = "48530278-0828-4a49-9772-0f3830dfa1e9"
 TensorOperations = "6aa20fa7-93e2-5fca-9bc0-fbd0db3c71a2"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
-
-[extras]
```

Comment:
Was this meant to be removed?

Reply:
I made this change to ask you if we should  allow users to run the command `Pkg.test("NDTensors"; test_args=["metal"])".
When `Metal` is under the `[extras]` and you run this test command the code fails with 
```
NDTensors: Error During Test at /Users/kpierce/.julia/packages/SafeTestsets/raUNr/src/SafeTestsets.jl:30
  Got exception outside of a @test
  LoadError: ArgumentError: Package Metal not found in current path.
  - Run `import Pkg; Pkg.add("Metal")` to install the Metal package.
  ```
  However, it looks like there is an issue in `compat` so I will remove it.

Reply:
Yeah, I guess ideally `Pkg.test("NDTensors"; test_args=["metal"])` would work (that seems like the only way to run the tests on Metal right now anyway?). If this is needed to allow that, I think it could make sense to do this, as long as it doesn't cause other issues like Metal not installing properly on certain systems.